src/components/dialog/dialog.js

```diff
@@ -294,10 +294,12 @@ function MdDialogDirective($$rAF, $mdTheming, $mdDialog) {
  *
  * @returns {obj} an `$mdDialogPreset` with the chainable configuration methods:
  *
- * - $mdDialogPreset#title(string) - sets title to string
- * - $mdDialogPreset#content(string) - sets content / message to string
- * - $mdDialogPreset#ok(string) - sets okay button text to string
- * - $mdDialogPreset#theme(string) - sets the theme of the dialog
+ * - $mdDialogPreset#title(string) - Sets the alert title.
+ * - $mdDialogPreset#textContent(string) - Sets the alert message.
+ * - $mdDialogPreset#htmlContent(string) - Sets the alert message as HTML. Requires ngSanitize
```

Comment:
`$sanitize` does not remove stuff like interpolation symbols. This means an unwary user could do:
```js
var e = document.createElement('p')
...
e.textContent(userName);
var htmlContent = e.innerHTML;
```

Then, a user could set their username to `{{ runEvilThing() }}` and it would get compiled and run.